src/DotNetProjectFile.Analyzers/Rule.cs

```diff
@@ -284,6 +284,16 @@ public static partial class Rule
         tags: ["clarity", "casing"],
         category: Category.Clarity);
 
+    public static DiagnosticDescriptor MigrateAwayFromBinaryFormatter => New(
+        id: 0032,
+        title: "Migrate away from BinaryFormatter",
+        message: "Migrate away from BinaryFormatter and disable <{0}>.",
```

Comment:
I'd argue that removing the line entirely should be preferred over disabling it, but perhaps that should be its own rule

Reply:
You're right. I've rephrased the message.